ocfweb/docs/docs/staff/procedures/accounts/alumni-reset.md

```diff
@@ -0,0 +1,62 @@
+[[!meta title="Alumni Account Reset"]]
+
+Occasionally former OCF members want to re-enable their accounts after
+they've been disabled for various reasons, including weak passwords,
+lack of CalNet UID, lack of Kerberos principal, etc.
+
+These are the relevant steps to take in various situations.
+
+#### Missing CalNet UID    {calnet}
+
+If an alumni cannot use the online password reset function to reset the
+password on their account, but they do have a CalNet login, you can manually
+add the Calnet UID to their account in LDAP to let them perform a password
+reset. They may also need to have a Kerberos principal added for them.
+
+Please make sure to confirm the user's CalNet ID using the Berkeley Directory.
```

Comment:
Alumni are often not listed in the Berkeley Directory, even if they have a CalNet login. I remember @daradib was able to confirm some identities via cal.berkeley.edu, maybe he can say more on that?